apps/src/templates/textResponses/textResponsesRedux.js

```diff
@@ -28,7 +28,7 @@ export const setTextResponses = (scriptId, responseData) => ({ type: SET_TEXT_RE
 export const startLoadingResponses = () => ({ type: START_LOADING_TEXT_RESPONSES });
 export const finishLoadingResponses = () => ({ type: FINISH_LOADING_TEXT_RESPONSES });
 
-export const asyncLoadTextResponses = (sectionId, scriptId, onComplete) => {
+export const asyncLoadTextResponses = (sectionId, scriptId, onComplete = () => {}) => {
```

Comment:
note because this line looks a little weird: we want to fire-and-forget this async method call in the new dashboard-served teacher dashboard, so i added a default `onComplete`